packages/workflow/tests/workflowConfigs.spec.js

```diff
@@ -0,0 +1,30 @@
+const settings = require('../settings');
+const baseConfig = require('../webpack.config')
+const prodConfig = require('../webpack.config.production')
+const profileConfig = require('../webpack.config.profile')
+
+const NOW = '2023-03-28T16:07:07.909Z';
+
+describe('webpack configs', () => {
+    beforeAll(async() => {
+        jest.spyOn(global.Date.prototype, 'toJSON').mockImplementation(() => NOW);
```

Comment:
this is to keep the new Date.toJSON() from the banner plugin from breaking the snapshots